game/scripts/vscripts/modules/panorama_shop/panorama_shop.lua

```diff
@@ -273,25 +273,57 @@ function PanoramaShop:PushItem(playerId, unit, itemName, bOnlyStash)
 		if unit == FindCourier(team) then
 			unit = hero
 		end
-		-- Stackable item abuse fix, not very good, but that's all I can do without smth like SetStackable
-		local hasSameStackableItem = item:IsStackable() and unit:HasItemInInventory(itemName)
-		if hasSameStackableItem then
-			Notifications:Bottom(playerId, {text="panorama_shop_stackable_purchase", style = {color = "red"}, duration = 4.5})
-		else
-			if not isInShop then SetAllItemSlotsLocked(unit, true, true) end
-			FillSlotsWithDummy(unit, false)
-			for i = DOTA_STASH_SLOT_1 , DOTA_STASH_SLOT_6 do
+		
+		-- Stackable item abuse fix
+		local SameStackableItemId = nil
```

Comment:
Use camel case for local variables: `sameStackableItemId`
Also don't assign nil to a variable, just use `local sameStackableItemId`